src/service/finder.ts

```diff
@@ -37,4 +56,13 @@ export default class Finder {
       },
     };
   }
+
+  private getAppIdFilter(appId: number): GetProgramAccountsFilter {
+    return {
+      memcmp: {
+        offset: JOB_ACCOUNT_LAYOUT.offsetOf("clientAppId"),
```

Comment:
@tin-snowflake this should be appId, not clientAppId.
appId is a pubkey, not number.

Reply:
Resolved!